lib/release-mgr/doc-tag.js

```diff
@@ -0,0 +1,10 @@
+const exec = require('../exec-promise');
+
+function changeDocTags (dir, tag) {
+  return exec(dir, [
+    ` grep -lR 'change-me' --include=\*.md doc/ \
```

Comment:
I tried many different other commands without success before finding this one so if you want to propose something else please test it in Node.js before